src/UseCases/CancelDonation/CancelDonationUseCase.php

```diff
@@ -59,17 +59,31 @@ public function cancelDonation( CancelDonationRequest $cancellationRequest ): Ca
 
 		$this->donationLogger->log( $donation->getId(), self::LOG_MESSAGE_FOR_BACKEND );
 
-		$this->sendConfirmationEmail( $donation );
+		try {
+			$this->sendConfirmationEmail( $donation );
+		}
+		catch ( \RuntimeException $ex ) {
+			return new CancelDonationResponse(
```

Comment:
We extracted creating the other response objects into a private method. Any specific reason why this is not the case here?